packages/dds/sequence/src/test/intervalIndexTestUtils.ts

```diff
@@ -68,3 +70,141 @@ export function generateRandomIntervals(options: RandomIntervalOptions) {
 
 	return intervals;
 }
+
+/**
+ * Infer the side of a SequencePlace.
+ */
+export function sideFromSequencePlace(place: SequencePlace): Side {
+	if (typeof place === "object") {
+		return place.side;
+	} else if (place === "start") {
+		return Side.After;
+	} else {
+		return Side.Before;
+	}
+}
+/**
+ * position identifies a character, and does not take into account which side of the character.
+ *
+ * @param place - The SequencePlace used as an expected position of a sequence endpoint.
+ *
+ * @param ifEnd - The position to expect if `place` is "end". This is not currently consistent.
```

Comment:
nit: naming is a bit confusing to me here, maybe posIfEnd would be better? The current name implies that the value would be a boolean stating whether or not the place is end (at least to me).